packages/parser/src/classMetadata.ts

```diff
@@ -1822,7 +1822,7 @@ const Paladin = {
       tags: [SpellTag.Defensive],
     },
     { spellId: '96231', name: 'Rebuke', tags: [] },
-    { spellId: '152262', name: 'Seraphim', tags: [] },
+    { spellId: '172321', name: 'Seraphim', tags: [] },
```

Comment:
is this a manual change? is it intentional?

Reply:
Yeah, 152262 is no longer the spell id for anything